### PR TITLE
Lokalise: Translations update (ht)

### PR DESCRIPTION
**Pull request by tstirrat**
This pull request was initiated by Lokalise (user Tim Stirrat) at 2020-04-24 01:43:31

### CODE REVIEW

**Comment by tstirrat** [comment]
Screen Shot 2020-04-23 at 4 49 38 PM
Screen Shot 2020-04-23 at 4 49 57 PM
Screen Shot 2020-04-23 at 4 50 22 PM
Screen Shot 2020-04-23 at 4 50 32 PM
Screen Shot 2020-04-23 at 4 50 50 PM
Screen Shot 2020-04-23 at 4 51 04 PM

File: app/locales/ht.json [hunk 1]
```diff
@@ -1,35 +1,54 @@
 {
+  "history": {
```
Comment (tremblerz) [inline app/locales/ht.json:2]: Are we using history key somewhere?

Comment (tstirrat) [reply app/locales/ht.json:2]: yep, in the exposure history screens.

Comment (tremblerz) [reply app/locales/ht.json:2]: my bad

**Pull request merged**